perceval/backend.py

```diff
@@ -100,32 +110,48 @@ def archive(self, obj):
     def categories(self):
         return self.CATEGORIES
 
+    @property
+    def classified_fields(self):
+        cfs = ['.'.join(cf) for cf in self.CLASSIFIED_FIELDS]
+        return cfs
+
     def fetch_items(self, category, **kwargs):
         raise NotImplementedError
 
-    def fetch(self, category, **kwargs):
+    def fetch(self, category, filter_classified=False, **kwargs):
         """Fetch items from the repository.
 
         The method retrieves items from a repository.
 
         :param category: the category of the items fetched
+        :param filter_classified: remove classified fields from the resulting items
         :param kwargs: a list of other parameters (e.g., from_date, offset, etc.
         specific for each backend)
 
         :returns: a generator of items
+
+        :raises BackendError: either when the category is not valid or
+            'filter_classified' and 'archive' are active at the same time.
         """
         if category not in self.categories:
             cause = "%s category not valid for %s" % (category, self.__class__.__name__)
             raise BackendError(cause=cause)
 
+        if filter_classified and self.archive:
```

Comment:
The archive stores the url, headers, payload and response of a HTTP request, and the filter is applied later. Why `filter_classified` is disabled when using the archive?

Reply:
Because the fields that are classified come on that response. Therefore, they will be stored in the archive which I think it doesn't make sense if you want to remove them from the result. It would cause security problems in your organization, if you don't remember there's an archive storing them.

Reply:
Perfect, could you add a coment on top of that condition ?

Reply:
I rather add it to the docstring of that method.